index.html

```diff
@@ -16,17 +16,27 @@
 </head>
 
 <body>
-  <header id="homepage">
-    <div> <a href="#homepage" class="logo">Bright Ezeamaka</a> </div>
+  <header>
+    <div class="logo">Bright Ezeamaka</div>
     <div class="dropdown">
-      <img src="./Images/Icon - Menu.png" alt="">
+      <img class="dropdown-img" src="./Images/Icon - Menu.png" alt="">
+    </div>
+    <div id="mobile-navigation">
+      <div class="remove-btn hidden">X</div>
+      <ul class="mobile-nav">
+        <li> <a href="#about-myself" class="mobile-item">Portfolio</a> </li>
+       
+        <li> <a href="#about-me" class="mobile-item">About</a></li>
```

Comment:
- [ ] Clicking on the About mobile menu option suppose to show the `about me` section, not showing the page from the beginning.

![chrome-capture-2023-0-4](https://user-images.githubusercontent.com/68796232/210449708-239f5101-f367-4fd4-b771-fd4ad1f1be3f.gif)